src/index.tsx

```diff
@@ -77,10 +74,15 @@ export const getResetPasswordTemplate = (
 
 export const getTransactionTemplate = (
   props: TransactionTemplateProps,
+  isResend?: boolean,
   options?: Mjml2HtmlOptions
```

Comment:
Sorry, I didn't notice in the last review, I think it is better to put `isResend` into props, so that `getXXXTemplate()` always take `props` and `options` as params

Reply:
ok! trying to implement it now